src/main/features/ccj/views/paid-amount-summary.njk

```diff
@@ -0,0 +1,95 @@
+{% extends "layout.njk" %}
+{% set title = 'Money claim' %}
+{% from "includes/macros/form.njk" import submitButton, csrfProtection %}
+
+{% block content %}
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <h1 class="heading-large">
+      <span class="heading-secondary">{{ t("County Court Judgment") }}</span>
+      {{ t("Amount to be paid by defendant") }}
+    </h1>
+
+    <div class="form-group">
+      <p>
+        {{ t("Amount payable by defendant is calculated based on the amount you're claiming and any other costs minus any amount paid by defendant.") }}
+      </p>
+      <table class="table-form">
+        <caption class="visuallyhidden">{{ t("Amount payable by defendant") }}</caption>
+        <thead>
+        <tr>
+          <th scope="col"></th>
+          <th class="numeric last" scope="col">{{ t("Amount") }}</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+          <th scope="row">{{ t("Claim amount") }}</th>
+          <td class="numeric last">
+            <span class="font-small">{{ claim.claimData.amount | numeral }}</span>
+          </td>
+        </tr>
+        {% if interestDetails %}
+        <tr>
+          <th scope="row">
+            <details role="group">
+              <summary role="button" aria-controls="details-content-0" aria-expanded="false">
+                {{ t("Interest to date of judgment") }}
+              </summary>
+              <div class="panel text-small" id="details-content-0" aria-hidden>
+                {{ t("Interest calculated at %s&#37; for %s days", {
```

Comment:
A `"Interest calculated at %s%%; for %s days"` could be more readable and not require `| safe`.

Reply:
ok